From 00d593c131f59dac8b1104ac941ab1581732b21d Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Fri, 18 Oct 2024 09:34:15 +0200
Subject: [PATCH 1328/1695] clk: stm32mp25: rename RCC_USBTCCFGR register into
 RCC_UCPDCFGR

Rename this register to be aligned with the reference manual.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git dc101dc21e26a82a7a83d0b8b727420b1017a7eb clk: stm32mp25: rename RCC_USBTCCFGR register into RCC_UCPDCFGR]
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Change-Id: I9d520978e0ce053fe195f3f5e7ac9a3a1754e66c
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/412461
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/drivers/clk/clk-stm32mp25.c         |  2 +-
 core/drivers/rstctrl/stm32mp25_rstctrl.c |  2 +-
 core/include/drivers/stm32mp25_rcc.h     | 10 +++++-----
 3 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/core/drivers/clk/clk-stm32mp25.c b/core/drivers/clk/clk-stm32mp25.c
index ba0d46cfe..927380a95 100644
--- a/core/drivers/clk/clk-stm32mp25.c
+++ b/core/drivers/clk/clk-stm32mp25.c
@@ -639,7 +639,7 @@ static const struct gate_cfg gates_mp25[GATE_NB] = {
 	GATE_CFG(GATE_USB3DR,		RCC_USB3DRCFGR,		1,	0),
 	GATE_CFG(GATE_USB3PCIEPHY,	RCC_USB3PCIEPHYCFGR,	1,	0),
 	GATE_CFG(GATE_PCIE,		RCC_PCIECFGR,		1,	0),
-	GATE_CFG(GATE_USBTC,		RCC_USBTCCFGR,		1,	0),
+	GATE_CFG(GATE_USBTC,		RCC_UCPDCFGR,		1,	0),
 	GATE_CFG(GATE_ETHSWMAC,		RCC_ETHSWCFGR,		1,	0),
 	GATE_CFG(GATE_ETHSW,		RCC_ETHSWCFGR,		5,	0),
 	GATE_CFG(GATE_ETHSWREF,		RCC_ETHSWCFGR,		21,	0),
diff --git a/core/drivers/rstctrl/stm32mp25_rstctrl.c b/core/drivers/rstctrl/stm32mp25_rstctrl.c
index 84092b555..bc10fb40b 100644
--- a/core/drivers/rstctrl/stm32mp25_rstctrl.c
+++ b/core/drivers/rstctrl/stm32mp25_rstctrl.c
@@ -179,7 +179,7 @@ const struct stm32_reset_cfg *stm32mp25_reset_cfg[STM32MP25_LAST_RESET] = {
 	RST(USB2PHY2_R,		RCC_USB2PHY2CFGR,	0),
 	RST(USB3DR_R,		RCC_USB3DRCFGR,		0),
 	RST(USB3PCIEPHY_R,	RCC_USB3PCIEPHYCFGR,	0),
-	RST(USBTC_R,		RCC_USBTCCFGR,		0),
+	RST(USBTC_R,		RCC_UCPDCFGR,		0),
 	RST(ETHSW_R,		RCC_ETHSWCFGR,		0),
 	RST(SDMMC1_R,		RCC_SDMMC1CFGR,		0),
 	RST(SDMMC1DLL_R,	RCC_SDMMC1CFGR,		16),
diff --git a/core/include/drivers/stm32mp25_rcc.h b/core/include/drivers/stm32mp25_rcc.h
index 526af8f0e..e8c9f326c 100644
--- a/core/include/drivers/stm32mp25_rcc.h
+++ b/core/include/drivers/stm32mp25_rcc.h
@@ -426,7 +426,7 @@
 #define RCC_USB3DRCFGR				U(0x808)
 #define RCC_USB3PCIEPHYCFGR			U(0x80C)
 #define RCC_PCIECFGR				U(0x810)
-#define RCC_USBTCCFGR				U(0x814)
+#define RCC_UCPDCFGR				U(0x814)
 #define RCC_ETHSWCFGR				U(0x818)
 #define RCC_ETHSWACMCFGR			U(0x81C)
 #define RCC_ETHSWACMMSGCFGR			U(0x820)
@@ -1969,10 +1969,10 @@
 #define RCC_PCIECFGR_PCIELPEN			BIT(2)
 #define RCC_PCIECFGR_PCIESTPEN			BIT(4)
 
-/* RCC_USBTCCFGR register fields */
-#define RCC_USBTCCFGR_USBTCRST			BIT(0)
-#define RCC_USBTCCFGR_USBTCEN			BIT(1)
-#define RCC_USBTCCFGR_USBTCLPEN			BIT(2)
+/* RCC_UCPDCFGR register fields */
+#define RCC_UCPDCFGR_UCPDRST			BIT(0)
+#define RCC_UCPDCFGR_UCPDEN			BIT(1)
+#define RCC_UCPDCFGR_UCPDLPEN			BIT(2)
 
 /* RCC_ETHSWCFGR register fields */
 #define RCC_ETHSWCFGR_ETHSWRST			BIT(0)
-- 
2.47.2

